test/e2e/dynamic_provisioning_test.go

```diff
@@ -462,7 +462,7 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 			}, isMultiZone),
 		}
 		podWithSnapshot := testsuites.PodDetails{
-			Cmd: "grep 'hello world' /mnt/test-1/data",
```

Comment:
could you provide reason why use cat instead of grep? it may help in other case too?